R/checks-srr.R

```diff
@@ -218,3 +218,7 @@ srr_checks_to_md <- function (checks) {
        "")
 
 }
+
+report_file <- function(checks) {
+    Sys.getenv("PKGCHECK_TEST_REPORT_FILE", checks$srr$report_file)
```

Comment:
Not entirely happy about this but it's needed for the snapshot. :grimacing:

Reply:
That's fine, unless you want to switch to an option? As long as tests are only ever triggered from main process, and not somewhere else, it should be fine.